content/security/authentication.md

```diff
@@ -823,12 +824,19 @@ If the vast majority of your endpoints should be protected by default, you can r
 First, register the `JwtAuthGuard` as a global guard using the following construction (in any module):
 
 ```typescript
-providers: [
-  {
-    provide: APP_GUARD,
-    useClass: JwtAuthGuard,
-  },
-],
+@@filename(app.module.ts)
+import { Module } from '@nestjs/common';
+import { APP_GUARD } from '@nestjs/core';
+
+@Module({
+  providers: [
+    {
+      provide: APP_GUARD,
+      useClass: RolesGuard,
+    },
+  ],
+})
+export class AppModule {}
```

Comment:
That's the same code from the global guards example here: https://docs.nestjs.com/guards#binding-guards